src/test/java/teammates/test/cases/webapi/UpdateFeedbackResponseActionTest.java

```diff
@@ -301,6 +302,34 @@ protected void testAccessControl_noFeedbackResponse_inaccessible() {
         assertThrows(EntityNotFoundException.class, () -> getAction(nonExistParams).checkAccessControl());
     }
 
+    @Test
+    protected void testAccessControl_missingIntent_inaccessible() {
+
+        ______TS("missing intent response parameters");
+
+        loginAsStudent(student1InCourse1.getGoogleId());
+
+        String[] missingIntentParams = new String[] {
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID, "randomNonExistId",
```

Comment:
Use a valid ID for this other than randomNonExistId. Then you shouldn't get the EntityNotFoundException and should instead properly get NullHttpParameterException.